src/Components/AdvancedSearch/AdvancedSearch.stories.tsx

```diff
@@ -1,24 +1,44 @@
 import React from 'react';
 import { ComponentStory } from '@storybook/react';
-import { getDefaultStoryDecorator } from '../../Models/Services/StoryUtilities';
-import AdvancedSearchModal from './AdvancedSearchModal';
+import {
+    getDefaultStoryDecorator,
+    IStoryContext
+} from '../../Models/Services/StoryUtilities';
+import AdvancedSearch from './AdvancedSearch';
 import { IAdvancedSearchProps } from './AdvancedSearch.types';
+import MockAdapter from '../../Adapters/MockAdapter';
 
 const wrapperStyle = { width: '100%', height: '100vh', padding: 8 };
 
 export default {
     title: 'Components/AdvancedSearch',
-    component: AdvancedSearchModal,
+    component: AdvancedSearch,
     decorators: [getDefaultStoryDecorator<IAdvancedSearchProps>(wrapperStyle)]
 };
 
-type AdvancedSearchStory = ComponentStory<typeof AdvancedSearchModal>;
+type AdvancedSearchStory = ComponentStory<any>;
```

Comment:
```suggestion
type AdvancedSearchStory = ComponentStory<AdvancedSearch>;
```

Reply:
I get type errors when I try to do interactive stories, I'll leave the any since I see other stories doing the same thing as well.